exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
+  VERSION = 1
+  ## Do not use the functions map, concat, count, length, etc.
+  ## and other built in ruby methods to solve  these
+  ##
+  def self.arrays(arr)
+    counter = 0
+    arr.each do
+      counter += 1
+    end
+    counter
+  end
+
+  def self.reverser(arr)
+    answer = []
+    until arr.empty?
+      answer << arr.pop
+    end
+    answer
+  end
+
+  def self.concatter(arr1, arr2)
+    answer = []
+    arr1.each do |element|
+      answer << element
+    end
+    arr2.each do |element|
+      answer << element
+    end
+    answer
+  end
+
+  def self.mapper(arr)
+    answer = []
+    arr.each do |element|
+      answer << (element + 1)
+    end
+    answer
+  end
+
+  def self.filterer(arr)
```

Comment:
The markdown said not to use those methods. Our original example had used all 4 of those, but we felt we needed to re-write all of them to fit what was wanted better.
If I was supposed to use those methods, then it was because the markdown file wasn't very clear since it was only 1 sentence.